tests/phpunit/tests/Renderer/Story/HTML.php

```diff
@@ -181,8 +181,6 @@ public function test_add_poster_images() {
 		$rendered = $this->setup_renderer( $post );
 
 		$this->assertContains( 'poster-portrait-src=', $rendered );
-		$this->assertContains( 'poster-square-src=', $rendered );
```

Comment:
@spacedmonkey We also have some references to `poster-square` and `poster-landscape` in the edit-story folder.

Reply:
Good catch. Fixed now.

Reply:
I think there's one more -- in the `mediaResponseWithPdfAndImage.json` file.